plugins/guests/fedora/cap/nfs_client.rb

```diff
@@ -0,0 +1,13 @@
+module VagrantPlugins
+  module GuestFedora
+    module Cap
+      class NFSClient
+        def self.nfs_client_install(machine)
+          machine.communicate.tap do |comm|
+            comm.sudo("yum -y install nfs-utils nfs-utils-lib")
```

Comment:
The block is not necessary here, `machine.communicate.sudo("yum -y install nfs-utils nfs-utils-lib")` should be just fine.